src/content/changes/highlights/2023-05-05.md

```diff
@@ -0,0 +1,20 @@
+---
+date: 2023-05-05T14:00:00
+title: Highlights for the week ending May 5, 2023
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## Apps
+
+- [falco-app](https://github.com/giantswarm/falco-app) version [v0.5.2](https://github.com/giantswarm/falco-app/blob/main/CHANGELOG.md#052---2023-05-04) adds a new Kyverno `PolicyException` allowing falco to run in clusters enforcing `restricted` Pod Security Standards, and replaces a deprecated toleration label.
+- [kyverno-app](https://github.com/giantswarm/kyverno-app) version [v0.14.4](https://github.com/giantswarm/kyverno-app/blob/main/CHANGELOG.md#0144---2023-05-03) introduces a new policy limiting the namespaces where Kyverno `PolicyExceptions` may be created. By default, customer exceptions may be created only in the `policy-exceptions` namespace.
+- [security-bundle](https://github.com/giantswarm/security-bundle) version [0.14.1](https://github.com/giantswarm/security-bundle/blob/main/CHANGELOG.md#0141---2023-05-04) (and 0.14.0) includes new the versions of Falco, Kyverno, and Trivy Operator mentioned in this announcement. To make App configuration and diffs easier to work with in GitOps workflows, it also changes the way config values are passed to the bundled Apps: rather than passing a single multi-line string containing each App's configuration, all keys under the App's top-level key will be copied into the App's values. **This is a breaking configuration change.** Users must change all places where they override default values of a security-bundle App. This is  typically a one character change, an [example of which is available in our PR](https://github.com/giantswarm/security-bundle/pull/85/files) changing our sample.
```

Comment:
```suggestion
- [security-bundle](https://github.com/giantswarm/security-bundle) version [0.14.1](https://github.com/giantswarm/security-bundle/blob/main/CHANGELOG.md#0141---2023-05-04) (and 0.14.0) includes new the versions of Falco, Kyverno, and Trivy Operator mentioned in this announcement. To make App configuration and diffs easier to work with in GitOps workflows, it also changes the way config values are passed to the bundled Apps: rather than passing a single multi-line string containing each App's configuration, all keys under the App's top-level key will be copied into the App's values. **This is a breaking configuration change.** Users must change all places where they override default values of a security-bundle App. This is typically a one character change, an [example of which is available in our PR](https://github.com/giantswarm/security-bundle/pull/85/files) changing our sample.
```